tests/cms/utils/test_slug_utils.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING
+
+import pytest
+from django.core.exceptions import ValidationError
+from pytest_django.fixtures import SettingsWrapper
+
+from integreat_cms.cms.models.pages.page_translation import PageTranslation
+from integreat_cms.cms.models.regions.region import Region
+from integreat_cms.cms.utils.slug_utils import generate_unique_slug
+
+if TYPE_CHECKING:
+    from integreat_cms.cms.utils.slug_utils import SlugKwargs
+
+
+@pytest.mark.django_db
+def test_generate_unique_slug_fallback(
+    settings: SettingsWrapper, load_test_data: None
+) -> None:
+    """
+    Test whether the :meth:`~integreat_cms.cms.utils.slug_utils.generate_unique_slug_helper` function correctly uses the fallback property
+    when no slug is provided
+    """
+    settings.DEBUG = True
+    mock_object = Region.objects.get(slug="augsburg")
```

Comment:
```suggestion
    region = Region.objects.get(slug="augsburg")
```
I think when you query an object from the database, it's not a mocked object, but a real one^^ Sorry for being so nitpicky :see_no_evil:

Reply:
But you're also completely right. originally I tried to mock this, but it didn't work out. Will be changed to your suggestion.